src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -57,6 +57,13 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter('twig.form.resources', $config['form_themes']);
 
+        $container->setParameter('twig.date.format', $config['date']['format']);
+        $container->setParameter('twig.date.interval_format', $config['date']['interval_format']);
+        $container->setParameter('twig.date.timezone', $config['date']['timezone']);
+        $container->setParameter('twig.number_format.decimal_point', $config['number_format']['decimal_point']);
+        $container->setParameter('twig.number_format.decimals', $config['number_format']['decimals']);
+        $container->setParameter('twig.number_format.thousands_separator', $config['number_format']['thousands_separator']);
+
```

Comment:
All those parameters should be removed and instead, you should inject the value in the service directly. That's what we are doing now as much as possible to avoid polluting the parameters with information you don't need to access.